realm/build.gradle

```diff
@@ -12,8 +12,8 @@ buildscript {
         classpath 'com.novoda:gradle-android-command-plugin:1.3.0'
         classpath 'com.github.skhatri:gradle-s3-plugin:1.0.2'
         classpath 'org.kt3k.gradle.plugin:coveralls-gradle-plugin:2.4.0'
-        classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.4.5'
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7'
+        classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:3.1.1'
```

Comment:
I got some exception in `realm/realm-annotations-processor` project and I couldn't find a way to solve it.
I checked other working projects such as `grade-plugin`, they still use `build-info-extractor-gradle:3.1.1` and `gradle-bintray-plugin:1.6`.

Downgrading is the only way that I could find..